packages/webpack-cli/lib/utils/Compiler.js

```diff
@@ -37,8 +35,7 @@ class Compiler {
     }
 
     generateOutput(outputOptions, stats) {
-        this.output.generateRawOutput(stats, this.compilerOptions);
-        process.stdout.write('\n');
+        process.stdout.write(`${stats.toString(this.compilerOptions.stats)}\n`);
```

Comment:
Any specific reason not to use `logger.raw()` here?

Reply:
Dont think so, got what we had originally without any changes

Reply:
/cc @jamesgeorge007

Reply:
I think using `logger.raw()` would be better here. WDYT?

Reply:
/cc @anshumanv Why we change it? Can you explain?

Reply:
We didn't change it, it has been this way. Logger seems better, let's change 👍